### DISCUSSION TITLE
How pytorch_lightning is compatible with version 1.7 of pytorch

### DISCUSSION THREAD

**Discussion by xiaolang01** [Lightning-AI/pytorch-lightning#13131] [General]
How pytorch_lightning is compatible with version 1.7 of pytorch，I need the 1.7 version of the torch environment, but every time I install pytorch_lightning, it will automatically uninstall the lower version of torch for me

**Comment by akihironitta** [comment]
`pytorch-lightning==1.6` dropped support for `torch<=1.7`. If you need to use PyTorch v1.7, you need to downgrade your Lightning version to v1.5.

**Reply by xiaolang01** [reply]
OK thanks
